release/preview/centos7.3/docker/Dockerfile

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+FROM centos:7.3.1611 as installer-env
```

Comment:
we should let it float in the from tag as well, otherwise you need to setup dependabot on each image, your choice.